### PR TITLE
Add Momokai Aurora Image

### PR DESCRIPTION
**Pull request by peepeetee**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/momokai/aurora/readme.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 # Momokai Aurora
 
+![Momokai Aurora](https://i.imgur.com/A7iHqzA.jpg)
```
Comment (dunk2k) [inline keyboards/momokai/aurora/readme.md:3]:
```suggestion
![Momokai Aurora](https://i.imgur.com/A7iHqzAh.jpg)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/momokai/aurora/readme.md (hunk 1 above)
Comment (peepeetee) [reply keyboards/momokai/aurora/readme.md:3]: explain

Comment (dunk2k) [reply keyboards/momokai/aurora/readme.md:3]: Please see `readme.me` section of [PR checklist](https://docs.qmk.fm/#/pr_checklist)

Comment (peepeetee) [reply keyboards/momokai/aurora/readme.md]: got it, thanks!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**